concent_api/concent_api/settings/base.py

```diff
@@ -310,9 +310,6 @@
 # completing verification.
 ADDITIONAL_VERIFICATION_TIME_MULTIPLIER = 2.0
 
-# A global constant defining the cost of additional verification if provider requested it.
```

Comment:
shouldn't it be left here? At least in commented out form?

Reply:
:+1: done